src/BatchFormat/source.extension.vsixmanifest

```diff
@@ -15,19 +15,25 @@ Function: Remove, Sort, Using, Format Document, and more combinations.</Descript
     <InstalledByMsi>false</InstalledByMsi>
     <SupportedProducts>
       <VisualStudio Version="10.0">
-        <Edition>Ultimate</Edition>
         <Edition>Premium</Edition>
         <Edition>Pro</Edition>
+        <Edition>Ultimate</Edition>
       </VisualStudio>
       <VisualStudio Version="11.0">
-        <Edition>Ultimate</Edition>
         <Edition>Premium</Edition>
         <Edition>Pro</Edition>
+        <Edition>Ultimate</Edition>
       </VisualStudio>
       <VisualStudio Version="12.0">
-        <Edition>Ultimate</Edition>
+        <Edition>Community</Edition>
         <Edition>Premium</Edition>
         <Edition>Pro</Edition>
+        <Edition>Ultimate</Edition>
+      </VisualStudio>
+      <VisualStudio Version="14.0">
+        <Edition>Community</Edition>
+        <Edition>Enterprise</Edition>
+        <Edition>Pro</Edition>
       </VisualStudio>
```

Comment:
Note that in VS2015, Premium & Ultimate editions are merged into Enterprise: https://www.visualstudio.com/downloads/download-visual-studio-vs#d-community.